src/ToggleButton.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts" context="module">
+  import { capitalize } from "@app/utils";
+
+  export interface ToggleButtonOption<T> {
+    count?: number;
+    title?: string;
```

Comment:
I think we should be fine with using the capitalized `value` almost everywhere, do you have an example when we would use `title`?

Reply:
Yeah, could remove it for now.

Reply:
Actually, we need it. Because we're using enums in some places and their value doesn't match the tab title. 🤦